packages/medusa/src/services/__tests__/new-totals.ts

```diff
@@ -528,14 +529,17 @@ describe("New totals service", () => {
         )
       })
 
-      it("should compute the gift cards totals amount in a taxable region", async () => {
+      it("should compute the gift cards totals amount if gift cards have a tax rate", async () => {
```

Comment:
suggestion: `should compute the gift cards totals amount using the gift card tax rate`